pegasus/sites.v3/hourofcode.com/styles/050-theme.css

```diff
@@ -18,7 +18,7 @@ body {
 
 #pageheader {
   width: 100%; 
-  background-color: #000;
+  background-color: rgba(0, 178, 192, 0.5);
```

Comment:
This was originally inline CSS on the div; I simply moved it to the stylesheet. Presumably that shouldn't change the behaviour, but will definitely do more testing!